chart/openfaas/values.yaml

```diff
@@ -94,10 +94,20 @@ gateway:
 operator:
   image: ghcr.io/openfaasltd/faas-netes:0.4.23
   create: false
+  # Unnecessary when running a single replica of the gateway
+  leaderElection:
+    enabled: false
+  reconcileWorkers: 2
   resources:
     requests:
       memory: "120Mi"
       cpu: "50m"
+  # When set to true, pprof will be enabled, and the 
+  # service "faas-provider" will gain an extra port to
+  # expose the pprof endpoint, this cannot be used in production
+  # since it may bypass authentication, and should only be used
+  # for debugging purposes
+  pprof: false
```

Comment:
I think we should also set default values for `reconcileQPS` and `reconcileBurst`.